go/teams/service_helper.go

```diff
@@ -649,12 +649,22 @@ func AcceptInvite(ctx context.Context, g *libkb.GlobalContext, token string) err
 	return err
 }
 
-func ParseAndAcceptSeitanToken(ctx context.Context, g *libkb.GlobalContext, tok string) error {
+func ParseAndAcceptSeitanToken(ctx context.Context, g *libkb.GlobalContext, tok string) (wasSeitan bool, err error) {
 	seitan, err := GenerateIKeyFromString(tok)
 	if err != nil {
-		return err
+		g.Log.CDebugf(ctx, "GenerateIKeyFromString error: %s", err)
+		g.Log.CDebugf(ctx, "returning TeamInviteBadToken instead")
+		return false, libkb.TeamInviteBadTokenError{}
+	}
+	g.Log.CDebugf(ctx, "found seitan token")
+	err = AcceptSeitan(ctx, g, seitan)
+	if err != nil {
+		g.Log.CDebugf(ctx, "AcceptSeitan error: %s", err)
+		g.Log.CDebugf(ctx, "returning TeamInviteBadToken instead")
+		return true, libkb.TeamInviteBadTokenError{}
```

Comment:
This could be a network error or anything. You think masking it is worth it?

Reply:
I thought of that, but the other option is to check for everything coming back...This fixes the "DB Error" one that started this.

Reply:
Actually, maybe with the import of 2646 implemented it will come through ok...let me try.

Reply:
Ok, it works.  I'll remove this mask.